app/controllers/api/trainees/withdraws_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Api
+  module Trainees
+    class WithdrawsController < Api::BaseController
+      def create
+        if trainee.blank?
```

Comment:
I'm wondering if it would be better to just let `raise`'s happen and elegantly handle them in one call as a controller filter should they occur (responding with the correct error/message). That'll help us keep the logic clean in the controller methods

Reply:
Fixed